charts/orchestrator-infra/Chart.yaml

```
@@ -0,0 +1,15 @@
+charts.openshift.io/name: Orchestrator-Infra
+charts.openshift.io/provider: Red Hat Developer Hub Team
+charts.openshift.io/supportURL: https://github.com/redhat-developer/rhdh-chart/issues
```

```suggestion
charts.openshift.io/supportURL: https://issues.redhat.com/browse/RHIDP
```

I also realized that we should be pointing to JIRA instead since we purposely disabled GH issues in this repo. It is being fixed in the `backstage` charts as well (https://github.com/redhat-developer/rhdh-chart/pull/114)